core-bundle/assets/controllers/operations-menu-controller.js

```diff
@@ -97,13 +97,6 @@ export default class OperationsMenuController extends Controller {
     }
 
     isInteractive (el) {
-        if (el instanceof HTMLImageElement || el instanceof HTMLAnchorElement || el instanceof HTMLButtonElement) {
-            return true;
-        }
-
-        // Also check the parent element if el is not interactive
-        const parent = el.parentElement;
-
-        return parent instanceof HTMLImageElement || parent instanceof HTMLAnchorElement || parent instanceof HTMLButtonElement;
+        return el instanceof HTMLAnchorElement || el instanceof HTMLButtonElement || el instanceof HTMLInputElement || el?.closest('a, button, input');
```

Comment:
While this may work, I'm unsure about using el?.closest here due to the deep nesting in the dom. If neither conditions match, it traverses the dom up till it finds something.

Most likely a negligible performance hit as this is only ever checked when opening a context menu but wanted to mention it.

Reply:
that's the point, we don't know if the operation has multiple `span` or whatever inside. We want to know if whatever is clicked is a link or a button, regardless of how deep it is nested.